sdcm/sct_config.py

```diff
@@ -137,6 +137,9 @@ class SCTConfiguration(dict):
         dict(name="cloud_credentials_path", env="SCT_CLOUD_CREDENTIALS_PATH", type=str,
              help="""Path to your user credentials. qa key are downloaded automatically from S3 bucket"""),
 
+        dict(name="cloud_cluster_id", env="SCT_cloud_cluster_id", type=int,
```

Comment:
where is it used?

Reply:
In the plugin, this replace all other cloud related params, and the need to use the iOS of the node, the new plugin would find them using this id